$OpenBSD: patch-Source_Core_Core_HW_WiimoteReal_IOhidapi_cpp,v 1.2 2020/03/08 02:15:51 thfr Exp $

change hid_init to hidapi_hid_init which is renamed locally
to avoid conflict with usbhid

Index: Source/Core/Core/HW/WiimoteReal/IOhidapi.cpp
--- Source/Core/Core/HW/WiimoteReal/IOhidapi.cpp.orig
+++ Source/Core/Core/HW/WiimoteReal/IOhidapi.cpp
@@ -41,7 +41,7 @@ namespace WiimoteReal
 {
 WiimoteScannerHidapi::WiimoteScannerHidapi()
 {
-  int ret = hid_init();
+  int ret = hidapi_hid_init();
   ASSERT_MSG(WIIMOTE, ret == 0, "Couldn't initialise hidapi.");
 }
 
